npm/ls-lint/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "ls-lint",
+  "version": "1.4.2",
+  "description": "The 64-bit binary for ls-lint",
+  "repository": "https://github.com/loeffel-io/ls-lint",
+  "license": "MIT",
+  "main": "./bin/index.js",
+  "bin": {
+    "ls-lint": "./bin/index.js"
+  },
+  "os": [
+    "darwin",
+    "linux",
+    "win32"
+  ],
+  "cpu": [
+    "x64"
+  ],
+  "scripts": {
+    "postinstall": "node postinstall.js"
```

Comment:
would love to rename: 

`postinstall.js` to `install.js`
`index.js` to `cli.js`